flytekit/core/tracker.py

```diff
@@ -216,6 +284,13 @@ def get_absolute_module_name(self, path: str, package_root: typing.Optional[str]
 _mod_sanitizer = _ModuleSanitizer()
 
 
+def _task_module_from_callable(f: Callable):
```

Comment:
can we type hint this and add a small comment for what the three things are?   also why is this only for Callable?  because all tasks are callable?